src/components/IRI.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { ReactComponent as Copy } from "../assets/copy_small.svg";
+import { Box, Button, Tooltip, Typography } from "@material-ui/core";
+import { IriItem } from "./SearchLabel";
+
+const Content = React.forwardRef((props: any, ref: any) => {
```

Comment:
It's better to avoid the TS `any` whenever possible. Here for example the props could be `ButtonProps`, which is a type that can be imported from `@material-ui/core` as far as I remember.

Reply:
Also I am not entirely sure if the `forwardRef` is really needed here.